MapboxDirections/MBRoadClasses.swift

```diff
@@ -46,6 +46,9 @@ extension RoadClasses: CustomStringConvertible {
         if contains(.ferry) {
             descriptions.append("ferry")
         }
+        if contains(.none) {
+            descriptions.append("none")
```

Comment:
Is `none` a valid value for the `exclude` URL query parameter? If not, omit this line so nothing gets added to the `descriptions` array.